scout/adapter/mongo/query.py

```diff
@@ -717,6 +717,16 @@ def secondary_query(self, query, mongo_query, secondary_filter=None):
             if criterion == "alt_count":
                 mongo_secondary_query.append({"tumor.alt_depth": {"$gt": query.get("alt_count")}})
 
+            if criterion == "somatic_score":
+                mongo_secondary_query.append(
+                    {
+                        "$or": [
+                            {"somatic_score": {"$gt": query.get("somatic_score")}},
+                            {"somatic_score": {"$exists": False}},
```

Comment:
Bit this way you return also those without somatic score, shouldn't this be True instead? 

So if you have a case without somatic score and you filter using the score then you don't get any variants, but if the case has somatic score then you get only variants with scopre higher than the threshold

Reply:
Why would you want that?

Reply:
From the text above: "We filter away low scoring variants, keeping non existing calls to leave variants only found by callers other than e.g. manta that have it.". At least to me, this is the way. Compare GnomAD etc. We very rarely want to remove variants from triage on missing indication, only if we have some kind of clear indication like high observed frequency, high observed CADD etc. Or a low observed somatic score. But not no-call.

Reply:
I mean if you want those with a score higher than a number, shouldn't it be

```
"$and": [
                            {"somatic_score": {"$gt": query.get("somatic_score")}},
                            {"somatic_score": {"$exists": True}},
```

Reply:
Nah, firstly gt kind of implies exists, secondly then the feature could be used to exclude calls from callers that dont implement somatic_score. That would not be very neat for the clinical side, although I'm sure some researchers would be happy.

Reply:
I don't know, to me it feels weird to filter by somatic score and get variants that have no somatic score at all..

Reply:
Huh, compare gnomAD, loqus counts, swegen, CADD?

Reply:
But I understand that you might want the filtering to be conservative. In this case perhaps explain in the tooltip that returned variants have score higher than filter or no score at all?

Reply:
> Huh, compare gnomAD, loqus counts, swegen, CADD?

I know, but they are also a bit weird for me :)